### ISSUE TITLE
Allow using the generator function on a HTML page directly

### ISSUE THREAD

**Issue by prantlf** [uuidjs/uuid#345]
The UUID generator can be built to a browser application using browserify, webpack, rollup or similar other bundler. However, if the application is built from pure JavaScript modules or [AMD] modules, this package cannot be used without an additional bundling work. (Scripts published from the NPM package cannot be loaded by the `script` element on a web page.)

How about including [UMD] modules in the NPM package, so that browser applications could load them directly, without each of them building such module themselves internally?

For example, by importing the UUID generator function globally:

```html
<script src="https://unpkg.com/uuid@3.4.0/browser/v1.min.js"></script>
<script>
  var id = uuidv1();
  console.log(id);
</script>
```

Another example, requiring the UUID generator as an [AMD] module (using [RequireJS], for example):

```html
<script src="https://unpkg.com/requirejs@2.3.6/require.js"></script>
<script>
  require.config({
    paths: {uuidv4: 'https://unpkg.com/uuid@3.4.0/browser/v4.min'}
  });
  require(['uuidv4'], function(uuid) {
    var id = uuid();
    console.log(id);
  });
</script>
```

[AMD]: https://github.com/amdjs/amdjs-api/blob/master/AMD.md#amd
[RequireJS]: https://requirejs.org/
[UMD]: https://github.com/umdjs/umd#umd-universal-module-definition

**Comment by ctavan** [comment]
@prantlf I'm considering to add a UMD build when releasing the v4.0.0 version of this module so that this library could be loaded directly through the various CDNs that just mirror npm package content.

Until then could you check if https://github.com/kelektiv/node-uuid/issues/302#issuecomment-524827935 works for you?

**Comment by ctavan** [comment]
This has been merged to the next branch with #357 and will be released with the upcoming major version release.

**Comment by ctavan** [comment]
We have just released [`uuid@7.0.0`](https://github.com/uuidjs/uuid/blob/v7.0.0/CHANGELOG.md#700-2020-02-24) which now features direct UMD builds again: https://github.com/uuidjs/uuid#umd-builds

Please feel free to open a new issue if you encounter any issues with the new version.
